composables/massmint/parsers/parseCsv.ts

```diff
@@ -1,70 +1,69 @@
+import type { OpenSeaAttribute } from '@kodadot1/hyperdata'
 import type { Entry } from './common'
 import { isValidEntry, removeQuotes, validFileExtension } from './common'
 
-function processCsvLine(line: string, csvHeaders: string[]): Partial<Entry> {
-  const values = line.split(',').map(value => removeQuotes(value.trim()))
-  const entry: Partial<Entry> = {
-    file: undefined,
-    name: undefined,
-    description: undefined,
-    price: undefined,
-  }
-
-  for (const [j, header] of csvHeaders.entries()) {
-    const value = values[j]?.trim()
-    if (header === 'price') {
-      entry.price = parseFloat(value) || undefined
-    }
-    else {
-      entry[header] = value || undefined
-    }
-  }
-
-  return entry
-}
+const DEFAULT_CSV_HEADERS = ['file', 'name', 'description', 'attributes', 'price']
 
-function parseCsvWithHeaders(lines, csvHeaders): Record<string, Entry> {
+function processCsvLines(lines: string[], csvHeaders: string[]): Record<string, Entry> {
   const entries: Record<string, Entry> = {}
 
   for (const line of lines) {
-    const entry = processCsvLine(line.trim(), csvHeaders)
+    const values = line.split(',').map(value => removeQuotes(value.trim()))
+    const entry: Partial<Entry> = {
+      file: undefined,
+      name: undefined,
+      description: undefined,
+      attributes: undefined,
+      price: undefined,
+    }
+
+    for (const [j, header] of csvHeaders.entries()) {
+      const value = values[j]?.trim()
+      if (header === 'price') {
+        entry.price = parseFloat(value) || undefined
+      }
+      else if (header === 'attributes') {
+        if (value) {
+          const attributes: OpenSeaAttribute[] = []
+          const attrs: Record<string, string> = {}
+          value.split(';').forEach((attr) => {
```

Comment:
move to constant on the top of the file `const ATTRIBUTE_SEPARATOR = ';'`

Reply:
updated